Automattic/wp-calypso
# Fix/migrate enzyme to react testing library - d3-base component

#### Proposed Changes

* Migrate the d3-base component from Enzyme to React testing library

#### Testing Instructions
 
1. Run the test for d3-base component with yarn run test-client:watch from source directory
2. Select P and then type `d3-base`
3. We should see that the correct test file was executed and run like this:
 ![](https://d.pr/i/TuGlQc+)
[Direct Link](https://d.pr/i/TuGlQc+)

Related to #63409
